docs/workloads/azure/backend/java/testing_java.md

```diff
@@ -169,20 +169,20 @@ The generated report can be viewed under `<PROJECT-NAME>/target/site/jacoco/inde
   <TabItem value="unix">
 
   ```bash
-  ./mvnw clean install -Powasp-dependency-check"
+  ./mvnw clean install -Powasp-dependency-check
   ```
 
   </TabItem>
   <TabItem value="windows">
 
   ```bash
-  mvnw.cmd clean install -Powasp-dependency-check"
+  mvnw.cmd clean install -Powasp-dependency-check
   ```
 
   </TabItem>
  </Tabs>
 
-The generated report can be viewed under `<PROJECT-NAME>/target/dependency-check.html`.
+The generated report can be viewed under `./target/dependency-check.html`.
 
 #### Mutation testing
 
```

Comment:
This requires clarification about what it is and how its being done.  This appears to come out of nowhere and is not present in any of the other workloads.

Reply:
If this is an optional thing that can be done then it should move into a section below as an option that can be used as it strictly is not mandatory.